cinema/models.py

```diff
@@ -84,23 +84,39 @@ class Ticket(models.Model):
     row = models.IntegerField()
     seat = models.IntegerField()
 
+    @staticmethod
+    def validate_ticket(
+            row: int,
+            seat: int,
+            movie_session: MovieSession,
+            error_to_raise
```

Comment:
The parameter `error_to_raise` is unconventional for a validation method. Typically, validation methods directly raise `ValidationError` without passing an error class as a parameter. Consider removing this parameter and directly raising `ValidationError`.